### DISCUSSION TITLE
Dva在umi4 还推荐用吗，

### DISCUSSION THREAD

**Discussion by wangsanshao** [umijs/umi#12387] [Q&A]
dva取消了subscriptions。
umi Max 还有内置valtio 数据流，这两种有什么建议方案，或者最佳实践，不知道该选哪个

**Comment by fz6m** [comment]
新项目最好使用现代数据流：管理全局状态使用 [jotai](https://github.com/pmndrs/jotai) / [valtio](https://umijs.org/docs/max/valtio) / [zustand](https://github.com/pmndrs/zustand) 管理（喜欢用哪个看个人，它们的运作方式和用法都有较大区别），管理请求状态使用 [react-query](https://umijs.org/docs/max/react-query) / [swr](https://github.com/vercel/swr) 。

现在 umi 4 里的 model / dva 等插件其实是为了 umi 3 升级而保留的上古功能，这些功能很久都不维护了，并且和现代数据流相比运行速度较慢，但有的人不需要在意应用运行速度，用习惯了还喜欢用，所以还是有用户会用这些功能的，但基于现代前端技术发展的角度，不建议继续使用，应该使用上文提到的现代数据流的库。

**Reply by keene2** [reply]
jotai 有案例么？如何全局组织 store
